apps/src/componentLibrary/segmentedButtons/segmentedButtons.module.scss

```diff
@@ -0,0 +1,218 @@
+@import "color";
+@import "@cdo/apps/componentLibrary/common/styles/mixins";
+
+.segmentedButtons {
+  display: flex;
+  align-items: flex-start;
+  gap: -1px;
+
+  .segmentedButton {
+    &:first-child {
+      border-start-start-radius: 4px;
+      border-end-start-radius: 4px;
+    }
+
+    &:last-child {
+      border-start-end-radius: 4px;
+      border-end-end-radius: 4px;
+    }
+
+    &:not(:last-child) {
+      margin-right: -1px;
+    }
+  }
+
+  // Selected styles
+  .selectedSegmentedButton {
+    background-color: $light_primary_500;
+    border-color: $light_primary_500;
+    color: $light_white;
+
+    // Hover styles
+    &:hover {
+      background-color: $light_primary_500;
+      border-color: $light_primary_500;
+      color: $light_white;
+    }
+
+    // Focus styles
+    &:focus-visible {
+      background-color: $light_primary_100;
+      border-color: $light_gray_200;
+      color: $light_black;
+    }
+
+    // Pressed styles
+    &:active {
+      background-color: $light_primary_500;
+      border-color: $light_primary_500;
+      color: $light_white;
+    }
+
+    // Disabled styles
+    &:disabled {
+      background-color: $light_primary_100;
+      border-color: $light_primary_100;
+      color: $light_white;
+    }
+  }
+}
+
+.segmentedButton {
+  display: flex;
+  justify-content: center;
+  align-items: baseline;
+  gap: 0.5rem;
+
+  background-color: $light_white;
+  border: 1px solid $light_gray_200;
+  border-radius: 0;
+  color: $light_black;
+  margin: 0;
+
+  // Hover styles
+  &:hover {
+    box-shadow: none;
+    background-color: $light_gray_100;
+    border-color: $light_gray_200;
+  }
+
+  // Focus styles
+  &:focus-visible {
+    outline: none;
+    background-color: $light_primary_100;
+    border-color: $light_gray_200;
+  }
+
+  // Pressed styles
+  &:active {
+    background-color: $light_primary_700;
+    // Important here is needed to rewrite line 3320, app/assets/stylesheets/application.scss */
```

Comment:
Yep, it's for 3328 line.
I would like to try that, but I would rather create a ticket for that and do it as a separate ticket since that might cause a lot of eyes diffs potentially and I want this PR to only affect design system component

Reply:
Oh yep for sure should be a different change 🙂 I've just been seeing it pop up a lot recently.

Reply:
Created a quick win ticket for this: https://codedotorg.atlassian.net/browse/DESIGN2-88